tutorcodejail/plugin.py

```diff
@@ -0,0 +1,48 @@
+from glob import glob
+import os
+import pkg_resources
+
+from .__about__ import __version__
+
+templates = pkg_resources.resource_filename(
+    "tutorcodejail", "templates"
+)
+
+config = {
+    "add": {
+        "SECRET_KEY": "{{ 24|random_string }}",
+    },
+    "defaults": {
+        "VERSION": __version__,
+        "HOST": "codejailservice",
+        "DOCKER_IMAGE": f"docker.io/ednxops/codejailservice:{__version__}",
+    },
+    "set":{
+        "SANDBOX_PYTHON_VERSION": "3.5.10",
```

Comment:
I suggest moving this to the "defaults" section. Then, use it as `CODEJAIL_SANDBOX_PYTHON_VERSION`. When possible, we should really try to avoid "set" entries. They are only required to override existing tutor settings.

Reply:
Done